src/main/java/com/google/cloud/genomics/dataflow/pipelines/AnnotateVariants.java

```diff
@@ -141,21 +143,17 @@ public AnnotateVariants(OfflineAuth auth,
 
   @Override
   public void processElement(
-      DoFn<SearchVariantsRequest, KV<String, VariantAnnotation>>.ProcessContext c) throws Exception {
+      DoFn<StreamVariantsRequest, KV<String, VariantAnnotation>>.ProcessContext c) throws Exception {
     Genomics genomics = GenomicsFactory.builder().build().fromOfflineAuth(auth);
 
-    SearchVariantsRequest request = c.element();
+    StreamVariantsRequest request = StreamVariantsRequest.newBuilder(c.element())
+        .addAllCallSetIds(callSetIds)
+        .build();
     LOG.info("processing contig " + request);
-    Iterable<Variant> varIter = FluentIterable
-        .from(Paginator.Variants.create(genomics, ShardBoundary.Requirement.STRICT).search(
```

Comment:
Yes.  There are two relevant commits in this PR (1) add the integration test to check existing functionality (2) update to streaming confirming that integration test still passes.